.github/workflows/ci.yml

```
@@ -0,0 +1,23 @@
+name: Node.js CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x, 12.x, 14.x]
```

I don't think we need to test all of these, only LTS should be supported.

```suggestion
        node-version: [16.x, 18.x, 20.x]
```